modin/core/execution/dask/implementations/pandas_on_dask/partitioning/partition.py

```diff
@@ -53,7 +51,8 @@ def __init__(self, data, length=None, width=None, ip=None, call_queue=None):
         self._length_cache = length
         self._width_cache = width
         self._ip_cache = ip
-        self._identity = uuid.uuid4().hex
+        # TODO: align with other engines
+        self._identity
```

Comment:
Doesn't this cause an error?
```python
>>> class Test:
...     def __init__(self):
...             self._a
...
>>> Test()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<stdin>", line 3, in __init__
AttributeError: 'Test' object has no attribute '_a'
```

Reply:
Here property is called from the base class

Reply:
Okay, but anyway, why do we need to call this at constructor?

Reply:
It doesn't really make sense, except that it's constantly in plain sight and should annoy us more to make these changes. I want to leave it as it is.

Reply:
Can you elaborate why we call it for Dask engine but don't call for others?

Reply:
We also call it for other engines when we create a message for `log.debug`. It just looks like a rudiment for Dask, but it's done on purpose.
```python
        log = get_logger()
        if self._is_debug(log):
            log.debug(
                "Partition ID: {}, Height: {}, Width: {}, Node IP: {}".format(
                    self._identity,  # <- call for Ray engine
                    ...
```

Reply:
I see, but the TODO doesn't look clear to me. Let's create an issue and put it here in the TODO to have more info on what should be done.

Reply:
Perhaps, but I plan to do it myself, that's enough for me.

Reply:
short-term perspective, yes